tests/core/sprout_checks/test_check_properties_sprout.py

```diff
@@ -0,0 +1,160 @@
+from pathlib import Path
+
+from pytest import fixture, mark, raises
+
+from seedcase_sprout.core.checks.check_error import CheckError
+from seedcase_sprout.core.properties import PackageProperties, ResourceProperties
+from seedcase_sprout.core.sprout_checks.check_properties import check_properties
+from seedcase_sprout.core.sprout_checks.failed_check_error import FailedCheckError
+from seedcase_sprout.core.sprout_checks.get_blank_value_for_type import (
+    get_blank_value_for_type,
+)
+from seedcase_sprout.core.sprout_checks.required_fields import (
+    PACKAGE_SPROUT_REQUIRED_FIELDS,
+    RESOURCE_SPROUT_REQUIRED_FIELDS,
+)
+
+
+@fixture
+def properties():
+    return PackageProperties(
+        name="package-1",
+        id="abc1",
+        title="Package 1",
+        description="A package.",
+        version="1.0.0",
+        created="2024-05-14T05:00:01+00:00",
+        licenses=[{"name": "a-license"}],
+        contributors=[{"title": "a contributor"}],
+        sources=[{"title": "a source"}],
+        resources=[
+            ResourceProperties(
+                name="resource-1",
+                path=str(Path("resources", "1", "data.parquet")),
+                title="Resource 1",
+                description="A resource.",
+            ),
+            ResourceProperties(
+                name="resource-2",
+                path=str(Path("resources", "2", "data.parquet")),
+                title="Resource 2",
+                description="A second resource.",
+            ),
+        ],
+    ).compact_dict
+
+
+@mark.parametrize("check_required", [True, False])
+def test_passes_full_properties(properties, check_required):
+    """Should pass if all required fields are present and correct."""
+    assert check_properties(properties, check_required=check_required) == properties
```

Comment:
Naming of tests :)) 

Could also be something like:

```suggestion
def test_check_passes_full_properties(properties, check_required):
    """Should pass if all required fields are present and correct."""
    assert check_properties(properties, check_required=check_required) == properties
```

Reply:
I realllllly don't mind, but there's an argument that "check" is implied in all test names because it's the only function we're testing.